src/modules/integrator-guide/pages/api-gw/index.adoc

```diff
@@ -86,17 +86,15 @@ When you click on the btn:[new] button, a new screen will appear where you need
 ----
 
 * *Endpoint* defines the path for the endpoint,
-* *method* defines what HTTP method the endpoint will listen for.
-//Editor's note: There should be better expressions for "listen for" unless it's a profi expression.
+* *method* defines the HTTP method for the endpoint.
 
 The endpoint will be accessible under `$BASE_URL/api/gateway$YOUR_ENDPOINT`.
-To exemplify; the `/test-endpoint` endpoint for the `https://latest.cortezaproject.org` instance is available on the hyperlink `https://latest.cortezaproject.org/api/gateway/test-endpoint`.
+To exemplify; the `/test-endpoint` endpoint for the `https://latest.cortezaproject.org` instance is available as `https://latest.cortezaproject.org/api/gateway/test-endpoint`.
 
 [TIP]
 ====
-You can specify different HTTP methods on the same route, but they need to be specified as different endpoints, ie. **GET** /customer and **PUT** /customer being two different integration gateway endpoints.
+You can specify different HTTP methods on the same route, but they need to be specified as different endpoints, ie. `*GET* /customer` and `*PUT* /customer` being two different integration gateway endpoints.
 ====
-//Editor's note: Any reason for capitalization of GET and PUT?
```

Comment:
These things (HTTP verbs) are generally capitalized so they should be yes

Reply:
Lovely, now I know :).